lib/zone.ts

```diff
@@ -141,6 +141,16 @@ interface Zone {
    */
   get(key: string): any;
   /**
+   * Returns a Zone which defines a `key`.
+   *
+   * Recursively search the parent Zone until a Zone which has a `key` is found.
+   * otherwise.
+   *
+   * @param key The key to retrieve.
```

Comment:
you are not retrieving a key. `The key to use for identification of the returned zone.`